ledger-core/application/testwalletapi/handlers.go

```diff
@@ -80,7 +80,12 @@ func (s *TestWalletServer) Create(w http.ResponseWriter, req *http.Request) {
 		TraceID:   traceID,
 		Error:     "",
 	}
-	defer func() { s.mustWriteResult(w, result) }()
+	defer func() {
+		if len(result.Error) != 0 {
+			logger.Error(result.Error)
```

Comment:
better to have whole result here, or at lest traceID

Reply:
Logger contains traceId field
It's error path. There is no result here